oggm/core/flowline.py

```diff
@@ -2433,10 +2433,11 @@ def run_from_climate_data(gdir, ys=None, ye=None, min_ys=None, max_ys=None,
         except AttributeError:
             ys = gdir.rgi_date
         # The RGI timestamp is in calendar date - we convert to hydro date,
-        # i.e. 2003 becomes 2004 (so that we don't count the MB year 2003
-        # in the simulation)
+        # i.e. 2003 becomes 2004 if hydro_month is not 1 (January)
```

Comment:
This is a bit arguable - in the alps for example the image is from Aug 2003 -> is it better to start from Jan 2003 or Jan 2004? Although it's not super important, I'd argue for 2004...

Reply:
ok, I understand. Then ys+=1 makes  sense independent of the hydro date conversion and I will add your comment just to make that clear